### PR TITLE
Enable sockets in CentOS package

### PR DESCRIPTION
**Pull request by gordonwang0**
Enables service sockets during postinstall.

### CODE REVIEW

**Comment by arsing** [comment]
You don't want to do it with the preset file thing?

**Comment by gordonwang0** [comment]
I figured adding the enable to an existing file was easier. If you think the preset file is better, I'll change it.

**Comment by arsing** [comment]
The preset file looks to be the same amount of work (four lines in `/usr/lib/systemd/system-preset/50-aziot-identity-service.preset` ) and appears to be "technically" more correct. My RPM-using distro doesn't ship any preset files and all services need to be enabled manually, but based on Fedora's docs it seems to be the preferred method. Dropping files is nicer that runtime commands and all that.

File: Makefile [hunk 1]
```diff
@@ -269,6 +269,9 @@ rpm:
 	mkdir -p $(RPMBUILDDIR)/SPECS
 	sed -e 's/@version@/$(PACKAGE_VERSION)/g; s/@release@/$(PACKAGE_RELEASE)/g' contrib/centos/aziot-identity-service.spec >$(RPMBUILDDIR)/SPECS/aziot-identity-service.spec
 
+	# Copy preset file to be included in the package
+	cp contrib/centos/00-aziot.preset $(RPMBUILDDIR)
```
Comment (gordonwang0) [inline Makefile:273]: @arsing I wasn't exactly sure where to put this; if you know a more suitable place, please suggest.

Comment (arsing) [reply Makefile:273]: Extra source files like patches or this go in SOURCES/, next to the regular tarball.

Comment (gordonwang0) [reply Makefile]: Ok, moved it.

**Review by arsing — APPROVED** [review]

**Review by varunpuranik — APPROVED** [review]

**Pull request merged**